colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -308,4 +306,9 @@ extends Controller[O,I](codec, ControllerConfig(config.pendingBufferSize, config
       worker ! Kill(id.get, DisconnectCause.TimedOut)
     }
   }
+
+  private def failRequest(s: SourcedRequest, exception: Throwable): Unit = {
+    errors.hit(tags = hpTags + ("type" -> exception.getClass.getName.replaceAll("[^\\w]", "")))
```

Comment:
An alternative would be to use `e.getClass.getSimpleName`. If you want to keep this code, where would be a good common place?